### PR TITLE
Update deliverables section with instructions for submitting files

### PR DESCRIPTION
**Pull request by lindsaymarkward**

### CODE REVIEW

**Comment by lindsaymarkward** [comment]
Hi @  HsuMyatMonWin

**Comment by lindsaymarkward** [comment]
Hi @HsuMyatMonWin - can you please review this and in particular, check my SRP. Thanks.

File: prac_03/README.md [hunk 1]
```diff
@@ -533,6 +533,13 @@ You may also like to do one or more of the GitHub courses on [LinkedIn Learning]
 # Deliverables
 This section summarises the expectations for marking in this practical.
 
+- Do not zip up your files.
```
Comment (lindsaymarkward) [inline prac_03/README.md:536]: Why not use caps for NOT, or make it **bold** :)

File: prac_05/README.md [hunk 2]
```diff
@@ -337,6 +337,13 @@ Now on to more practice programming...
 
 This section summarises the expectations for marking in this practical.
 
+- Do not zip up your files.
+- Please submit each file separately.
+- Ensure each file has the correct/exact name, including the extension.
+- Ensure your code is not commented-out (only comments should be commented).
+
+Files required:
```
Comment (HsuMyatMonWin) [inline prac_05/README.md:345]: Missing "wimbledon.py" in files required.

**Review by HsuMyatMonWin — COMMENTED** [review]
No SRP issues as far as I can see and instructions are clear.

File: prac_05/README.md (hunk 2 above)
Comment (lindsaymarkward) [reply prac_05/README.md:345]: Hi. Sorry, I used your name in the mention for a seminar, as it was at the top of the list. I did not really want you to review this PR from 12 months ago :)

**Pull request merged**